.mercury/docs/decisions/DEC-2-taskbundle-lightweight.md

```
@@ -0,0 +1,55 @@
+| codeScope | implementationReceipt |
+| readScope | mainReview |
+| allowedWriteScope | handoffToAcceptance |
+| docsMustUpdate | reworkHistory (injected separately via buildReworkPrompt) |
+| docsMustNotTouch | linkedIssueIds |
+| definitionOfDone | originatorSessionId |
+| requiredEvidence | reviewConfig |
+| reworkCount, maxReworks | |
+
+## Analysis
+
+### Already lightweight
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)